pkg/wallet/wallet.go

```diff
@@ -37,8 +36,10 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
 	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
 	"github.com/hyperledger/aries-framework-go/pkg/kms"
+	"github.com/hyperledger/aries-framework-go/pkg/kms/localkms"
 	"github.com/hyperledger/aries-framework-go/pkg/store/connection"
 	"github.com/hyperledger/aries-framework-go/spi/storage"
+	"github.com/piprate/json-gold/ld"
```

Comment:
please revert this last import line and keep it under `"github.com/google/uuid"` as it was prior to this pr, this is not an AFGO import, so it must be part of third party dependencies block above AFGO's dependencies.

Reply:
this is still not reverted, please revert this line with the removed line 18 above